src/script/script.h

```diff
@@ -184,6 +184,9 @@ enum opcodetype
     OP_DETERMINISTICRANDOM = 0xc0,
     OP_CHECKSIGFROMSTACK = 0xc1,
     OP_CHECKSIGFROMSTACKVERIFY = 0xc2,
+    OP_SHA256_INITIALIZE = 0xc4,
+    OP_SHA256_UPDATE = 0xc5,
+    OP_SHA256_FINALIZE = 0xc6,
```

Comment:
Note that using two underscores in op name would mean breaking (useless?) convention that there's always just one underscore in `OP_<NAME>`.